app/models/list.rb

```diff
@@ -0,0 +1,7 @@
+class List < ActiveRecord::Base
+  belongs_to :therapist
+  has_many :request
```

Comment:
You have to pluralize the table when doing `has_many`, i.e. `has_many :requests`